css/main.css

```diff
@@ -4,6 +4,20 @@
    Author's custom styles
    ========================================================================== */
 
+ 
+/* sprint 1 Img zoom on Hover */
+ 
+img {
+   transition: transform .2s; /* Animation */
+   margin: 0 auto;
+ 
+}
+ 
+img:hover {
+   transform: scale(1.25);
```

Comment:
While the zoom makes the image 'pop', I would consider reducing the scale. The image when hovered will block the text:
![scrnsht](https://user-images.githubusercontent.com/35419335/89687505-36e6d880-d8b5-11ea-9a40-41f2183ac833.JPG)

```suggestion
   transform: scale(1.1);
```

Reply:
ahh, i see, okay will do